feat: allow support for ruby system locale and spec.default paths

malbajun opened the pull request:
- [x] + ruby system locale path used in find along with posix default path
- [x] + default path mostly used in alt linux tree
- [x] * test specs

puppetlabs-jenkins commented:
Can one of the admins verify this patch?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/puppetlabs/puppet?pullRequest=9562) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/puppetlabs/puppet?pullRequest=9562) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/puppetlabs/puppet?pullRequest=9562) it.</sub>

majioa committed:
feat: allow support for ruby system locale and spec.default paths

+ ruby system locale path used in find along with posix default path
+ default path mostly used in alt linux tree
* test specs
